src/mongo/MongoScrapbook.ts

```diff
@@ -112,7 +112,7 @@ async function executeCommand(activeEditor: vscode.TextEditor, database: MongoDa
 			throw new Error(`Error near line ${err.range.start.line}, column ${err.range.start.character}: '${err.message}'. Please check syntax.`);
 		}
 
-		if (command.name === 'find') {
+		if (command.name === 'find' && !command.chained) {
```

Comment:
Maybe add a comment in the code for this one, or just link to the bug. It's not immediately clear why `chained` matters without context